<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1649931800955' stop='1649931840936'>
    <name>HRM login Example</name>
    <title>HRM login Example</title>
    <test-cases>
        <test-case start='1649931830959' status='failed' stop='1649931835696'>
            <name>TC_001 - verify login with valid user credenatils</name>
            <title>TC_001 - verify login with valid user credenatils</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (C:\Users\Act\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at listOnTimeout (node:internal/timers:557:17)
                    at processTimers (node:internal/timers:500:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>